test-apps/presentation-test-app/package.json

```diff
@@ -6,7 +6,7 @@
     "name": "Bentley Systems, Inc.",
     "url": "http://www.bentley.com"
   },
-  "version": "0.0.0",
+  "version": "3.0.0-dev.69",
```

Comment:
what's the point of this change?

Reply:
same comment as for the other test, ill revert this.

Reply:
I see the version is still there